connect/mirror/src/test/java/org/apache/kafka/connect/mirror/integration/MirrorConnectorsIntegrationBaseTest.java

```diff
@@ -101,8 +101,9 @@
  */
 @Tag("integration")
 public class MirrorConnectorsIntegrationBaseTest {
+    private static final Object TOPIC_LOCK = new Object();
```

Comment:
Can you explain what this lock is accomplishing? There's only one test running at a time within the suite.